VideoWeb/VideoWeb/ClientApp/src/app/security/auth.guard.ts

```diff
@@ -15,7 +14,7 @@ export class AuthGuard implements CanActivate {
                 console.log('AuthorizationGuard, canActivate isAuthorized: ' + isAuthorized);
 
                 if (!isAuthorized) {
-                    this.router.navigate([`/${pageUrls.IdpSelection}`]);
+                    this.router.navigate(['/login']);
```

Comment:
Should maybe used ``` `/${pageUrls.Login}` ``` instead.

Reply:
Reinstated the last change. anyway will be reverted back to ${pageUrls.IdpSelection}, soon after 1.26 release.